hbase/versions.py

```diff
@@ -11,7 +11,7 @@
         "phoenix": "5.2.0",
         "hbase_profile": "2.4",
         "hadoop": "3.3.6",
-        "jmx_exporter": "1.0.1-stackable",  # update the stackable/jmx/config<version> folder too
+        "jmx_exporter": "1.1.0",  # update the stackable/jmx/config<version> folder too
```

Comment:
note: HBase versions still need updating

Reply:
Are you able to elaborate? Both versions are up to date according to the spreadsheet:
- 2.4.18
- 2.6.1

Reply:
Nevermind, they have been bumped in 3124bdc856adcdb6c4ff89262dc712fc24f9f2e2 already.
Sorry!

Reply:
This is not resolved. The folder needed to be renamed. This is causing the following build error: https://github.com/stackabletech/docker-images/actions/runs/13808117261/job/38629092041#step:4:558

Reply:
Fixup PR: https://github.com/stackabletech/docker-images/pull/1024